client/lib/fifo/fifo_windows.go

```diff
@@ -0,0 +1,69 @@
+package fifo
+
+import (
+	"io"
+	"net"
+
+	winio "github.com/Microsoft/go-winio"
+)
+
+const PipeBufferSize = int(^uint16(0))
+
+type FIFO struct {
```

Comment:
Let's not export it so the Window's version doesn't export a different API from the non-Window's version.